app/services/articles/feed.rb

```diff
@@ -38,5 +38,10 @@ def default_home_feed(user_signed_in: false)
       end
       [featured_story, hot_stories]
     end
+
+    def default_home_feed(user_signed_in: false)
+      _featured_story, stories = default_home_feed_and_featured_story(user_signed_in: user_signed_in)
```

Comment:
At first I thought it would make more sense to just return hot stories from `default_home_feed` and have `default_home_feed_and_featured_story` call that and then get the featured story as well, but I see why you're doing it. It's to avoid doing another query correct?

Reply:
Yeah, mostly: the "featured" story is simply the first hot story.